.changelog/26227.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:new-resource
+aws_networkmanager_vpc_attachment
+```
+
+```release-note:new-resource
+aws_networkmanager_attachment_acceptor
```

Comment:
Should be named `aws_networkmanager_attachment_accepter` 😄.

Reply:
https://english.stackexchange.com/questions/317122/difference-between-accepter-and-acceptor

Other similar resource use _er_:

```console
% grep accept internal/provider/provider.go 
			"aws_detective_invitation_accepter": detective.ResourceInvitationAccepter(),
			"aws_dx_hosted_private_virtual_interface_accepter": directconnect.ResourceHostedPrivateVirtualInterfaceAccepter(),
			"aws_dx_hosted_public_virtual_interface_accepter":  directconnect.ResourceHostedPublicVirtualInterfaceAccepter(),
			"aws_dx_hosted_transit_virtual_interface_accepter": directconnect.ResourceHostedTransitVirtualInterfaceAccepter(),
			"aws_directory_service_shared_directory_accepter": ds.ResourceSharedDirectoryAccepter(),
			"aws_ec2_transit_gateway_peering_attachment_accepter":  ec2.ResourceTransitGatewayPeeringAttachmentAccepter(),
			"aws_ec2_transit_gateway_vpc_attachment_accepter":      ec2.ResourceTransitGatewayVPCAttachmentAccepter(),
			"aws_vpc_endpoint_connection_accepter":                 ec2.ResourceVPCEndpointConnectionAccepter(),
			"aws_vpc_peering_connection_accepter":                  ec2.ResourceVPCPeeringConnectionAccepter(),
			"aws_guardduty_invite_accepter":            guardduty.ResourceInviteAccepter(),
			"aws_macie2_invitation_accepter":                 macie2.ResourceInvitationAccepter(),
			"aws_ram_resource_share_accepter": ram.ResourceResourceShareAccepter(),
			"aws_securityhub_invite_accepter":            securityhub.ResourceInviteAccepter(),
```

Reply:
I'm happy to make the change to be in line with the other resources in the provider. For the record, google will correct you based on Oxford's dictionary :)
![image](https://user-images.githubusercontent.com/3812162/184171697-1d671f7b-bc45-4494-9fc9-43426607ad51.png)

Reply:
upon further examination, i yield

😭